Bottom: 16d7e028be47236faf14d43083133eb5f32ca31f
Top:    cc5db995a4e554efb0b5734a24c367917d4a5d99
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-01-06 21:11:44 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
